.gitlab-ci.yml

```diff
@@ -54,7 +54,11 @@ fossa:
 
 .sign-docker:
   extends: .trigger-filter
+  allow_failure: true
```

Comment:
Do we need this allow_failure?

Reply:
Underlying signing service sometimes returns 500 randomly (already reported to its maintainers). I didn't want it to block deployments. On the other hand, we can always rety the job. I will remove it.